examples/Simple/Quicksort.hs

```diff
@@ -0,0 +1,58 @@
+{-# LANGUAGE LinearTypes #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+
+module Simple.Quicksort
```

Comment:
I'm missing some context. If this example is discussed in a post, a link to the post would be in order in the module documentation.

Reply:
The post isn't out yet. I added some doc at the top.